apps/dot-cdn/src/app/dotcdn.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from '@angular/core';
+import { ReplaySubject } from 'rxjs';
+import { CoreWebService, SiteService } from '@dotcms/dotcms-js';
+import { pluck, mergeMap } from 'rxjs/operators';
+import * as moment from 'moment';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class DotCDNService {
+    currentSite$: ReplaySubject<string> = new ReplaySubject(null);
```

Comment:
Why `ReplaySubject`? and also, if you need to be subscribe to the site change the site service have a method or property for that.

Reply:
Might have to change the approach here